frontend/lib/src/components/widgets/DataFrame/arrowUtils.ts

```diff
@@ -240,7 +240,19 @@ export function getColumnFromArrow(
   data: Quiver,
   columnPosition: number
 ): BaseColumnProps {
-  const title = data.columns[0][columnPosition]
+  const columnHeaders = data.columns.map(column => column[columnPosition])
+  const title = columnHeaders[columnHeaders.length - 1]
```

Comment:
could you please leave a comment why the `title` is derived from the headers? I guess this is due to the arrow format, but its a little bit non-intuitive. Also, is this equivalent to `columnHeaders[0]` or can there be other information stored in `columnHeaders` than the `title` and actual headers?

Reply:
Also for my learning: is it possible to have arrow-serialized dataframes that do not have headers and `columnHeaders[columnHeaders.length - 1]` could use the wrong headers (the last item in the array if it resolves to `columnHeaders[-1]`?

Reply:
> could you please leave a comment why the title is derived from the headers? 

column headers just are just the names of the column. You probably refer to why this is from `data.columns`, which is indeed confusing .. and I have no clue why it was named this why. But this is something I'm planning to refactor in Quiver at some point.

Reply:
> serialized dataframes that do not have headers  
I think we have never seen this. But I added a fallback to an empty title if that happens.